api/src/main/java/org/asynchttpclient/BoundRequestBuilder.java

```diff
@@ -93,6 +97,16 @@ public BoundRequestBuilder setBody(String data) {
     }
 
     @Override
+    public BoundRequestBuilder setBody(Publisher<ByteBuffer> publisher) {
+        return super.setBody(new ReactiveStreamBodyGenerator(client.getConfig().getExecutorService(), publisher));
+    }
+
+    @Override
+    public BoundRequestBuilder setBody(BodyGenerator bodyGenerator) {
+        return super.setBody(bodyGenerator);
+    }
```

Comment:
I've done so for consistency with regard to other methods overridden in this same source. See this comment https://github.com/dotta/async-http-client/blob/issue/544-reactive-streams-support/api/src/main/java/org/asynchttpclient/BoundRequestBuilder.java#L49-L52 for context.